src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -12,7 +12,41 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal static class Extensions
     {
+        public static string CLREventProviderName = "Microsoft-Windows-DotNETRuntime";
+
         private static EventLevel defaultEventLevel = EventLevel.Verbose;
+        // Keep this in sync with runtime repo's clretwall.man
+        private static Dictionary<string, long> CLREventKeywords = new Dictionary<string, long>()
+        {
+            { "gc", 0x1 },
```

Comment:
You can make string comparisons case-insensitive, which would allow you to make these PascalCase.